src/lib.rs

```diff
@@ -11,3 +11,4 @@ pub mod tcp;
 pub mod udp;
 pub mod vlan;
 pub mod vxlan;
+mod llc;
```

Comment:
```suggestion
pub mod llc;
```

Otherwise not really usable. :slightly_smiling_face: